### ISSUE TITLE
Unable to set value from props

### ISSUE THREAD

**Issue by vicentedealencar** [alexkuz/react-input-enhancements#10]
I needed a button to set the state of a masked input and have some trouble. I see there is demo for `<Autocomplete />` but not to `<Mask />`, is it not trivial?

What I ended up doing to make it work was to change [that line](https://github.com/alexkuz/react-input-enhancements/blob/master/src/Mask.jsx#L120) to `if (value) {` and it got working. I am not sure of what bugs this change may cause, but I couldn`t get this block to run other way. 

Maybe I missed something.

**Comment by alexkuz** [comment]
Yeah, I haven't tested it as a controlled component, actually, it's quite possible it's not working properly.
